Client/Frontend/Browser/BrowserViewController/BrowserViewController+Wallet.swift

```diff
@@ -377,6 +419,66 @@ extension Tab: BraveWalletEventsListener {
   }
 }
 
+extension Tab: BraveWalletSolanaEventsListener {
+  func accountChangedEvent(_ account: String?) {
+    Task { @MainActor in
+      if let webView = webView {
+        let script: String
+        if let account = account {
+          script = "if (solanaWeb3) { window.solana.emit('accountChanged', new solanaWeb3.PublicKey('\(account)')) }"
```

Comment:
The address here and in `SolanaProviderScriptHandler` (comment below) comes from core so it _should_ be fine, but given other comments on this piece I think it would help to use the `htmlEntityEncodedString` to be safe.

Reply:
https://github.com/brave/brave-ios/pull/6050/commits/87993ce8e4d3855c9e0ac2f522d60c999d6d337f